docs/modules/ROOT/pages/how-tos/vsphere/pre-install-checklist.adoc

```diff
@@ -0,0 +1,175 @@
+
+[[fn2]]^2^: The bootstrap node is a temporary machine that can be deleted after the installation.
+
+[[fn3]]^3^: Number and dimensioning of the worker nodes is dependent on your workload.
+
+[[fn4]]^4^: RHCOS = Red Hat CoreOS
+
+== Network
+
+=== Address and DHCP
+
+The OpenShift VMs should get their IP address via DHCP. Create DHCP reservations with the specified MAC address of the VM.
+
+=== DNS records
+
+Several DNS records are required for OpenShift to work correctly.
+
+The https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-dns-user-infra_installing-vsphere-network-customizations[OpenShift documentation] lists the required records in detail.
+
+To illustrate, the following variables are set:
+
+* `<base_domain>`: `appuio.ch`
+* `<cluster_name>`: `ocp4-vs`
+
+[CAUTION]
+====
+`<cluster_name>` must match the folder name in vSphere, in which all OpenShift VMs reside, for vSphere storage integration to work correctly.
+====
+
+With the defined variables, the following DNS records are required:
+
+|===
+|Record |Type |Target 
+
+|`api.ocp4-vs.appuio.ch` |A/CNAME |Console Load Balancer
+
+|`api-int.ocp4-vs.appuio.ch` |A/CNAME |Console Load Balancer
+
+|`*.apps.ocp4-vs.appuio.ch` |A/CNAME |Apps Load Balancer
+
+|`console.ocp4-vs.appuio.ch` |CNAME |Apps Load Balancer
+
+|`etcd-0.ocp4-vs.appuio.ch` |A |Master 1
+
+|`etcd-1.ocp4-vs.appuio.ch` |A |Master 2
+
+|`etcd-2.ocp4-vs.appuio.ch` |A |Master 3
+
+|`_etcd-server-ssl._tcp.ocp4-vs.appuio.ch` |SRV |See https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-dns-user-infra_installing-vsphere-network-customizations[documentation] for details.
+|===
+
+[WARNING]
+====
+By default, Windows DNS servers create a SRV record for every A record.
+For the A records of `etcd`, make sure that the SRV records are not created (or delete them, if they were), so the hostnames can be enumerated correctly.
+====
+
+=== Certificates
+
+Certificates for the following hostnames need to be created:
+
+* `api.ocp4-vs.appuio.ch`
+* `console.ocp4-vs.appuio.ch`
+* `*.apps.ocp4-vs.appuio.ch` (wildcard certificate)
+
+[WARNING]
+====
+For the certificates to be valid in modern web browser, make sure the _Subject Alternative Name (SAN)_ contains the hostname of the certificate.
+
+[[additional-ca]]
+If the certificates are issued from your own private PKI, the RootCA certificate of your PKI needs to be added to the `install-config.yaml` as described in the https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#installation-configure-proxy_configuring-a-custom-pki[documentation]:
+[source,bash]
+----
+...
+additionalTrustBundle: | 
+    -----BEGIN CERTIFICATE-----
+    <MY_TRUSTED_CA_CERT_1>
+    -----END CERTIFICATE-----
+    -----BEGIN CERTIFICATE-----
+    <MY_TRUSTED_CA_CERT_2>
+    -----END CERTIFICATE-----
+...
```

Comment:
```suggestion
…
```

There is a character for this. Just placing three dots, typographically looks odd.